projects/scannet_offline_eval/demo/proto_robot/mock_spot.py

```diff
@@ -0,0 +1,168 @@
+import io
+import logging
+import os
+import pickle
+import random
+import sys
+
+import grpc
+import robodata_pb2
+import robodata_pb2_grpc
+import torch
+from PIL import Image
+from torchvision import transforms
+
+sys.path.append("../../../")
+import home_robot.mapping.voxel.voxel as v
+
+spot_filename = "/private/home/ssax/home-robot/src/home_robot/home_robot/datasets/robot/data/spot_/09_18_fre_spot/spot_output.pkl"
```

Comment:
Let's set up these files either using a hydra or click/argparse? 
Hydra should be easy and we can use [this](https://github.com/facebookresearch/home-robot/blob/e21e2cb582287f4aaa190928255ee49e87731d49/src/home_robot/home_robot/agent/multitask/sparse_voxel_instance_map.py#L215-L264) as an example

Reply:
added argparse